source/js/components/join/join.jsx

```diff
@@ -265,17 +267,34 @@ export default class JoinUs extends React.Component {
           {!this.state.apiSuccess ? `${this.props.ctaHeader}` : `Thanks!`}
         </h5>
         {!this.state.apiSuccess ? (
-          <p
+          <div
             dangerouslySetInnerHTML={{
               __html: this.props.ctaDescription
             }}
           />
         ) : (
-          <p
-            dangerouslySetInnerHTML={{
-              __html: this.props.thankYouMessage
-            }}
-          />
+          <React.Fragment>
+            <p>
+              If you haven’t previously confirmed your opt-in to a
+              Mozilla-related email subscription you may have to do so now.{" "}
```

Comment:
Does `&nbsp;` work rather than `{" "}`? If so, that'll make future localization a lot easier.

Reply:
Moved the strings to `locale-strings.jsx` and it seems like strings there are using `{" "}` already so I'm gonna keep the new strings that way.